src/components/MyKiva/MyKivaProfile.vue

```diff
@@ -48,16 +55,31 @@ const props = defineProps({
 		type: Boolean,
 		default: false,
 	},
+	userInHomepage: {
+		type: Boolean,
+		default: false,
+	},
 });
 
 const { lender, userInfo } = toRefs(props);
 
 const lenderName = computed(() => {
 	const firstName = userInfo?.value?.userAccount?.firstName ?? '';
 	const lastName = userInfo?.value?.userAccount?.lastName ?? '';
+
+	if (props.userInHomepage) {
+		return `${firstName}`;
+	}
+
 	return `${firstName} ${lastName}`;
 });
 
+const headerMsg = computed(() => {
+	return props.userInHomepage
+		? `For you, <u>${lenderName.value}</u>`
```

Comment:
In the screenshot, the underline isn't appearing, was the screenshot taken before this `<u>` was added?

Reply:
What is the local limitation? I haven't worked with the underlining locally in `ui`.

Reply:
Not sure, I'm OOO, but interested if we have a fix to show it locally.

Reply:
Interesting, good to know!